src/ex3/cliTasks.js

```diff
@@ -0,0 +1,94 @@
+import chalk from "chalk";
+import { Command } from "commander";
+import { ItemManage_CLI } from "./ItemManager_cli.js";
+
+const program = new Command();
+const itemManager = new ItemManage_CLI();
+
+program
+  .name("cli-tasks")
+  .version("1.0.0")
+  .description("CLI tasks manager for ex3");
+
+program
+  .command("add")
+  .alias("a")
+  .description("Add new task")
+  .argument("<task>", "Task to add")
+  .action(async (taskInput) => {
+    const isCompleted = false;
+    const isAdded = await itemManager.addTask(taskInput, isCompleted);
+    if (isAdded) {
+      console.log(chalk.green("Task added"));
+    } else {
+      console.log(chalk.red("Task not added"));
+    }
+  });
+
+program
+  .command("complete")
+  .alias("c")
```

Comment:
![image](https://user-images.githubusercontent.com/96736630/173380830-ee964c41-7bd2-4065-b92e-79a5decf7c51.png)
This command doesn't work, code crashes